test/unit/modules/api-translateSpec.js

```diff
@@ -36,11 +36,15 @@ describe('Unit: Testing api-translate', function () {
     describe('when translating with the service...', function () {
 
         it('should return the translated string if key exists', inject(function ($translate) {
-            $translate('SALUTE').should.be.equal('Hello');
+            $translate('SALUTE').then(function(translation) {
```

Comment:
$translate service has changed to return a Promise in latest version of `angular-translate` so minor or major version update needed.

Reply:
Agree on that.